_ext/jboss_developer.rb

```diff
@@ -411,6 +412,7 @@ def create_page(page, content)
       # Returns a Ruby Hash of the HTTP payload.
       def create_payload(page, content)
         path = create_path page
+        puts "Creating page #{path} from content #{page.output_path}"
```

Comment:
@Dantheman720 We can drop this line if we have the other two log statements I think.